scCompare/helpers.py

```diff
@@ -447,6 +449,10 @@ def derive_statistical_group_cutoff(
     Args:
         adata_map: Mapping dataset.
         cluster_key: `adata.obs` key to cluster by.
+        use_fisher (optional): Whether or not to use a fisher-transformed correlation to
+            derive a threshold for cutting off correlations. Default = `False`.
+        n_stdev (optional): Standard deviation cutoff if using fisher transformation.
+            Ignored if `use_fisher` = `False`. Default = 2.6.
         n_mad_floor (optional): Automatically calculate MAD, but can't be lower than
             `n_mad_floor`. If set to `None`, no lower bound. Default = 5.
         n_mad (optional): Use exactly this many MADs to calculate statistical cutoffs.
```

Comment:
Can you add a comment here that `n_mad` and `n_mad_floor` will be ignored if `use_fisher` is True?

Reply:
Done